sdk/dotnet/unity/unity.msbuildproj

```diff
@@ -9,7 +9,7 @@
 <Project>
     <PropertyGroup>
         <BaseOutputPath>bin/</BaseOutputPath>
-        <Configuration>Debug</Configuration>
```

Comment:
Do not do that. Instead, you should pass `/p:Configuration=Release` when you invoke MSBuild.